scripts/run_travis_tests.sh

```diff
@@ -20,12 +20,13 @@ mode=$1
 sparkVersion=2.4.4
 hadoopVersion=2.7
 
-if [ "$mode" = "unit" ];
-then
+if [ "$mode" = "unit" ]; then
   echo "Running Unit Tests"
   mvn test -DskipITs=true -B
-elif [ "$mode" = "integration" ];
-then
+elif [ "$mode" = "functional" ]; then
+  echo "Running Functional Test Suite"
+  mvn test -pl hudi-utilities -Pfunctional-test-suite -B
```

Comment:
to begin with utilities module; later expand to others

Reply:
let's make the PR title reflective of this scope?